dftimewolf/lib/collectors/gcloud.py

```diff
@@ -77,21 +78,25 @@ def Process(self):
   def SetUp(self,
             analysis_project_name,
             remote_project_name,
-            incident_id,
-            zone,
-            boot_disk_size,
-            boot_disk_type,
-            cpu_cores,
+            incident_id=None,
+            zone='us-central1-f',
+            create_analysis_vm=True,
+            boot_disk_size=50,
+            boot_disk_type='pd-standard',
+            cpu_cores=4,
```

Comment:
It's better to leave these arguments as positional args, so that they error out if the recipe is not providing everything it needs (a recipe developer will catch errors earlier on).

(in another PR) It might be good to change this across the whole codebase.

Reply:
will open an issue for this and do it in a separate PR in all places